alica_engine/include/engine/PlanBase.h

```diff
@@ -129,7 +129,7 @@ class PlanBase
     int _treeDepth;
     std::atomic<bool> _running;
     bool _sendStatusMessages;
-    bool _isWaiting;
+    std::atomic<bool> _isWaiting;
```

Comment:
why does this need to be atomic?

Reply:
We have an access to this variable from different threads.

Reply:
maybe I missed something, but this variable seems to be accessed only from within a critical section

Reply:
`bool isWaiting() const { return _isWaiting; }`